app/views/assignments/_language_skills_language_select.html.slim

```diff
@@ -7,5 +7,7 @@
   ul.dropdown-menu
     li
       = link_to t('all'), request_params_filter({language_skills_language_cont: ''})
+    - Client.first_languages.each do |s|
+      li= link_to s[0], request_params_filter({language_skills_language_cont: s[1]})
```

Comment:
You can also do `.each do |(name, code)|` to deconstruct the array items into their own variables.